wandb/sdk/service/streams.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env python
+"""wandb grpc server.
+
+- WandbServer:
+- StreamMux:
+- StreamRecord:
+- WandbServicer:
+
+"""
```

Comment:
Probably want to update the docstring here?

Reply:
Improved:
```
+"""streams: class that manages internal threads for each run.
 
+StreamThread: Thread that runs internal.wandb_internal()
+StreamRecord: All the external state for the internal thread (queues, etc)
+StreamAction: Lightweight record for stream ops for thread safety with grpc
+StreamMux: Container for dictionary of stream threads per runid
 """
```